hsv_param lower
hsv_param upper
hough_param circle
